docs/rancher/virtualization-management.md

```diff
@@ -63,11 +63,11 @@ The following example provides a good explanation of how the multi-tenant featur
 1. Create a VM with one of the images that you have uploaded.
 1. Log in with another user, e.g., `project-readonly`, and this user will only have the read permission of this project.
 
-!!!note
-    A known issue was found that allows the [read-only user to be able to manage API actions](https://github.com/harvester/harvester/issues/1406).
-
 ## Delete Imported Harvester Cluster
 
+!!! note
+    You will need to reset the `cluster-registration-url` to empty on the setting page of the imported Harvester cluster first. A known issue will be addressed via [#1686](https://github.com/harvester/harvester/issues/1686).
```

Comment:
This doesn't seem to be necessary. Removing the cluster from rancher dashboard should be enough

Reply:
IIRC, the Rancher will not delete the agent node pod on the downstream Harvester cluster, and rebooting the agent node will add the cluster back unexpectedly.

Reply:
No, it won't add the cluster back on restart. The cluster agent is in charge of the connection, it does not create clusters.

Reply:
cool, thanks for the clarification.